api/src/main/java/ca/bc/gov/educ/api/graduation/service/ReportService.java

```diff
@@ -96,10 +83,10 @@ public ReportData prepareReportData(
 		return data;
 	}
 
-	private List<NonGradReason> getNonGradReasons(List<GradRequirement> nonGradReasons) {
+	private List<NonGradReason> getNonGradReasons(List<ca.bc.gov.educ.api.graduation.model.dto.GradRequirement> nonGradReasons) {
```

Comment:
Code duplication: getNonGradReasons and getNonGradReasonsAchvReport (line 292) are identical. Can you remove one?

Reply:
Removed duplicate